Helper function for getting inverse model matrix in WGSL shaders

jakkos-net opened the pull request:
In 0.11 you could easily access the inverse model matrix inside a WGSL shader with `transpose(mesh.inverse_transpose_model)`. This was changed in 0.12 when `inverse_transpose_model` was removed and it's now not as straightfoward. I wrote a helper function for my own code and thought I'd submit a pull request in case it would be helpful to others.

alice-i-cecile commented:
Really excellent comments, thanks for that :)

eerii reviewed (approved):
Very useful, thanks! I checked on a sample project and this works wonderfully

jakkos-net commented:
Following a [conversation in the discord](https://discordapp.com/channels/691052431525675048/866787577687310356/1341838039244083201) I've fixed the merge conflicts, renamed the function to match the changed names of other functions, and tested that it still works :)

fjkorf reviewed (approved):
This code is great. Clean and well documented.
